.github/workflows/build-and-test.yaml

```diff
@@ -48,6 +48,8 @@ jobs:
 
           python -m pip install -r python/dev-requirements.txt
           python -m pip install ./python
+          python -m pip install ./python[flink]
```

Comment:
PyFlink and PySpark have some conflict dependencies like py4j. For now, this conflict would cause failure when running `setup.py` to build Feathub's python package without using `extras_require`, but there will be at most some warnings when we install Feathub along with both flink and spark dependencies, and run tests locally. I think we can run all the tests in one workflow for now, and divide them into seperate workflows if there are test failures in future.

Reply:
According to offline discussions, I'll modify Feathub's test and CI structure to compile, install and test different processor separately.